src/helpers/config.ts

```diff
@@ -123,6 +131,8 @@ exports.generateRedirects = async ({ netlifyConfig, basePath, i18n }) => {
       from: `${basePath}/*`,
       to: HANDLER_FUNCTION_PATH,
       status: 200,
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
```

Comment:
I left that in as I did the first pass to TS. I figured we'd want to get this in incrementally and worry too much about problematic missing types for now.

Reply:
Is this type missing in `@netlify/build` then?

Reply:
Nah just one that would require a bit of refactoring of the function this ts-ignore refers to